examples/wifi/expanded/requests_wifi_api_github.py

```diff
@@ -22,11 +20,12 @@
 # 900 = 15 mins, 1800 = 30 mins, 3600 = 1 hour
 sleep_time = 900
 
-try:
-    from secrets import secrets
-except ImportError:
-    print("Secrets File Import Error")
-    raise
+# Get WiFi details, ensure these are setup in settings.toml
+ssid = os.getenv("CIRCUITPY_WIFI_SSID")
+appw = os.getenv("CIRCUITPY_WIFI_PASSWORD")
```

Comment:
What is `appw` for?

Reply:
I just standardized in what there was the most of. Assumed it was an internal preference. Can change them all to `password` or anything else that makes sense.

Reply:
Ahhh- `AP PassWord`

Reply:
I'd prefer `password`. This is the only other change from me.